stop.sh

```diff
@@ -1,8 +1,6 @@
 #!/bin/bash
 
 echo "Shutting down Akeneo service"
-source .env
-export SOURCE_PATH="${WORKING_DIR}"
 docker-compose down
```

Comment:
do you really think that this stop script is needed? It is just hide things no? I will likely know how to use docker-compose but I don't know your interface on stop.sh =/   If I don't know I will read the README file which can point me to right direction.